default.nix

```diff
@@ -76,20 +80,7 @@ let
     cardanoLib = localLib.iohkNix.cardanoLib;
     daedalus-bridge = self.bridgeTable.${nodeImplementation};
 
-    nodejs = let
-      njPath = pkgs.path + "/pkgs/development/web/nodejs";
-      buildNodeJs = pkgs.callPackage (import (njPath + "/nodejs.nix")) {
-        python = pkgs.python3;
-        icu = pkgs.icu68; # can’t build against ICU 69: <https://chromium-review.googlesource.com/c/v8/v8/+/2477751>
-      };
-    in
-      buildNodeJs {
-        enableNpm = true;
-        version = "14.17.0";
-        sha256 = "1vf989canwcx0wdpngvkbz2x232yccp7fzs1vcbr60rijgzmpq2n";
-        patches = pkgs.lib.optional pkgs.stdenv.isDarwin (njPath + "/bypass-xcodebuild.diff");
-      };
-
+    nodejs = pkgs.nodejs-16_x;
```

Comment:
I missed when that happened :open_mouth: Hmmm

Reply:
`release/4.12.0` has an older version – the red fragment from `develop` is correct, IMO